backend/src/main/java/gov/cdc/usds/simplereport/idp/repository/OktaRepository.java

```diff
@@ -17,6 +17,16 @@
  * <p>Handles all user/organization management in Okta
  */
 public interface OktaRepository {
+  Integer OKTA_ORGS_LIMIT = 200;
```

Comment:
⚠️ Okta rate limits: https://developer.okta.com/docs/reference/rl-global-mgmt/

since we are limited to around 600 calls in one minute -- to err on the side of caution, I set the limit to 200 since we make 2 calls to the group endpoint per org